src/test/scala/org/ensime/intg/BasicWorkflow.scala

```diff
@@ -71,11 +71,8 @@ class BasicWorkflow extends FunSpec with Matchers {
         val javaSearchSymbol = project.rpcPublicSymbolSearch(List("java", "io", "File"), 2)
         javaSearchSymbol match {
           case SymbolSearchResults(List(
-            TypeSearchResult("java.awt.image.ImageObserver", "ImageObserver", 'class,
```

Comment:
@rorygraves I don't think so. It's a search call for at most 2 results that used to return three results, with the top result for java io file being java.awt.image.ImageObserver. It now returns just java.io.File, which is I assume what you'd want this search to return.